bokeh/command/subcommands/serve.py

```diff
@@ -102,7 +102,7 @@
 accept requests from ``localhost:<port>`` where ``<port>`` is the port
 that the server is configured to listen on (by default: {DEFAULT_PORT}).
 
-If an asterix ``*`` is used in the host value (for example ``--host *``) then
+If an asterix ``*`` is used in the host value (for example ``--host '*'``) then
```

Comment:
It's necessary to quote `*` in shell commands otherwise file globing will kick-in.